gridLang/GridLangConstants.ts

```diff
@@ -106,6 +106,23 @@ export const IntegerCellDef: CellDef = {
     parse: (value: any) => parseInt(value),
 }
 
+export const IntegersCellDef: CellDef = {
+    keyword: "",
+    cssClass: "IntegerCellDef",
+    description: "",
+    regex: /^-?[0-9 ]+$/,
+    requirementsDescription: `Must be an integer or a list of integers`,
+    valuePlaceholder: "12345 23456",
+    parse: (value: any) => {
+        if (typeof value === "number") return [value]
+        else
+            return value
+                .split(" ")
+                .map((item: string) => parseInt(item.trim()))
```

Comment:
There is a `parseIntOrUndefined` function Utils which might be a bit nicer here. Not important though, this also works well.